tests/unit/docker/test_hub_build.py

```diff
@@ -1,14 +1,20 @@
 from pathlib import Path
 
 import pytest
+import os
 
 from jina.docker.hubio import HubIO
 from jina.excepts import ImageAlreadyExists
 from jina.helper import yaml
+from jina.docker.helper import credentials_file
 from jina.parser import set_hub_build_parser, set_hub_list_parser, set_hub_pushpull_parser
 
 cur_dir = Path(__file__).parent
 
+@pytest.fixture
+def access_token_github():
+    token = os.environ['GITHUB_TOKEN']
```

Comment:
```suggestion
    token = os.environ,get('GITHUB_TOKEN', None)
```
Please add a way to error out, if env is not set